src/Microsoft.AspNetCore.Razor.Language/Syntax/Syntax.xml

```diff
@@ -0,0 +1,102 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+
+<Tree Root="SyntaxNode">
+  <PredefinedNode Name="SyntaxToken" Base="SyntaxNode" />
+  
+  <!-- Common -->
+  <AbstractNode Name="RazorNodeSyntax" Base="SyntaxNode" />
```

Comment:
RazorNodeSyntax vs RazorSyntaxNode? I think I like the latter much better.

Reply:
I could be convinced though

Reply:
Actually, `RazorNodeSyntax` is a base class for `CSharpNodeSyntax` and `HtmlNodeSyntax`. Since, these are all abstract classes, I don't see why they should be named similar to `SyntaxNode`. I'll change the names.